dev/ci/user-overlays/README.md

```diff
@@ -21,14 +21,11 @@ and the branch name), then a `.sh` extension (`[0-9]{5}-[a-zA-Z0-9-_]+.sh`).
 Example: `13128-SkySkimmer-noinstance.sh` containing
```

Comment:
The text above about the file containing very simple logic should probably have been updated as well.